src/sentry/testutils/pytest/fixtures.py

```diff
@@ -207,7 +200,6 @@ def default_user(factories):
     return factories.create_user(email="admin@localhost", is_superuser=True)
 
 
-@pytest.mark.django_db
```

Comment:
for context: marks on fixtures do nothing -- this is a new warning in pytest 8.x